js/app.js

```diff
@@ -0,0 +1,107 @@
+console.log('yo')
+// combine objects, arrays, and functions
+const curbArray = [
+	characters: {
```

Comment:
assuming that this is #4, you don't want a key for the object. (i.e. delete `characters:`)